src/test/rpc/AMMInfo_test.cpp

```diff
@@ -75,6 +75,52 @@ class AMMInfo_test : public jtx::AMMTestBase
             }
         });
 
+        // Invalid parameters *and* invalid LP account, default API version
+        testAMM([&](AMM& ammAlice, Env&) {
+            Account bogie("bogie");
+            std::vector<std::tuple<
+                std::optional<Issue>,
+                std::optional<Issue>,
+                std::optional<AccountID>,
+                bool>>
+                vals = {
```

Comment:
`vals` is the same in a few tests. Consider declaring it once outside of `testAMM` and then using it in each test.